.github/workflows/ci.yml

```diff
@@ -15,7 +15,7 @@ jobs:
         image: [ubuntu-20.04, ubuntu-22.04]
     runs-on: ${{ matrix.image }}
     env:
-      CC: ${{ matrix.compiler }}
+      CC: ${{ matrix.compiler }} -static
```

Comment:
I think you should be able to do this via configure somehow, there is `--enable-all-static` which seems to use ld flag `-all-static` which i don't think i've used myself before.

Reply:
glibc can be a pain to build static with but jq i hope can be fine as i think the most pain comes from programs that need NSS or DNS

Reply:
Updated to use the `--enable-static` & `--enable-all-static` flags for LInux & MacOS, Windows already does that: https://github.com/jqlang/jq/pull/2620/commits/2480dd2522cfbdf36dd1b7be287b82b74bb07cf3

Reply:
Hmm...tests failed for ubuntu 22.04 after static is enabled: https://github.com/jqlang/jq/actions/runs/5432682781. They failed for "Segmentation fault", e.g. https://github.com/jqlang/jq/suites/14001000347/artifacts/781225809

Reply:
That's not good :( maybe revert for now? i can digg into it later if you want

Reply:
Please help :). I disabled static build for ubuntu 20.04 for now: https://github.com/jqlang/jq/pull/2620/commits/65202425d3bbf62795a50ae20d5c10a536ab0a24. Feel free to enable it once it's fixed.